libdokan/fs.go

```diff
@@ -370,6 +377,20 @@ func (f *FS) open(ctx context.Context, oc *openContext, ps []string) (dokan.File
 			return nil, false, dokan.ErrAccessDenied
 		}
 		return f.root.private.open(ctx, oc, ps[1:])
+	case `TEAM` == ps[0]:
```

Comment:
Not necessarily related to this PR, but I was hoping `TEAM`, `PRIVATE`, and `PUBLIC` would be defined in libdokan/common.go like their lower-cased friends :)